data_analysis/notebooks/businesses.ipynb

```diff
@@ -3,9 +3,20 @@
   {
    "cell_type": "code",
```

Comment:
It might be more convenient to build a dictionary in this function instead of a tuple. That way it isn't necessary to remember how the return values were ordered. Something like

```
def socials_from_soup(socials, classes):
    return {
        c: socials.find(class_=c).text
        for c in classes
        if socials.find(class_=c)
    }
```

You can use `d.get` on the returned dictionary to handle the case where a class was not found.

Reply:
Good catch on the facebook, I had originally used is not, but must've changed and copy pasted it for some reason. I did change it to a dictionary, which does work much cleaner.